change.go

```diff
@@ -8,6 +8,7 @@ func Change(schema interface{}, changes ...map[string]interface{}) *Changeset {
 
 	if len(changes) > 0 {
 		ch.changes = changes[0]
+		ch.ignorePrimary = true
```

Comment:
I think `Change` is okay, it's just like PutChange, changed field need to be specified explicitly using map 🤔

Reply:
Ok, now it is more clear - I will remove the flag for this function